web/crux-ui/Dockerfile

```diff
@@ -1,24 +1,16 @@
-# stage 1, dep install
-FROM node:19.2-alpine AS DEPS
-WORKDIR /app
-
-RUN apk add --no-cache libc6-compat nasm autoconf automake bash libltdl libtool gcc make g++ zlib-dev
-COPY package.json package-lock.json ./
-RUN npm ci --no-fund
-
-# stage 2, build app
-FROM node:19.2-alpine AS BUILDER
+# stage 1, build app
+FROM docker.io/library/node:19.2-alpine AS BUILDER
 WORKDIR /app
 
 ENV NODE_ENV production
 ENV NEXT_TELEMETRY_DISABLED 1
 
-COPY --from=DEPS /app/node_modules ./node_modules
 COPY . .
+RUN npm ci --no-fund
```

Comment:
We still need to copy `package-lock.json` if we want to use `npm ci` install.

Reply:
Nevermind missed the `COPY . .`

Reply:
would be nice to copy the package-lock.json in a different layer, so we can reuse those layers while building an image if the dependencies are unchanged.

Reply:
Totally forgot about those! WIll do. Thanks!

Reply:
Fixed. Also carried this change over to crux. @m8vago please check!